monitoring/mongodb/alerts.test.yaml

```diff
@@ -333,23 +333,30 @@ tests:
   - name: MongoDbRSNotSynced
     interval: 1m
     input_series:
-      - series: mongodb_rs_members_state{namespace="zenko", rs_nm="rs0", pod="data-db-mongodb-sharded-mongos-0", member_state="SECONDARY"}
-        values: 0x10
-      - series: mongodb_rs_members_state{namespace="zenko", rs_nm="rs0", pod="data-db-mongodb-sharded-mongos-1", member_state="SECONDARY"}
-        values: 0x10
-      - series: mongodb_rs_members_state{namespace="zenko", rs_nm="rs0", pod="data-db-mongodb-sharded-mongos-2", member_state="SECONDARY"}
-        values: 1x10
+      - series: mongodb_rs_members_state{cl_id="65d9fdecfdcca86bb7cd2b98", container="metrics", endpoint="metrics", instance="10.233.178.160:9216", job="zenko/data-db-mongodb-sharded-shard0-data", member_idx="data-db-mongodb-sharded-shard0-data-0.data-db-mongodb-sharded-headless.zenko.svc.cluster.local:27017", member_state="PRIMARY", namespace="zenko", pod="data-db-mongodb-sharded-shard0-data-0", prometheus="metalk8s-monitoring/prometheus-operator-prometheus", rs_nm="data-db-mongodb-sharded-shard-0", rs_state="1"}
+        values: 2x10
+      - series: mongodb_rs_members_state{cl_id="65d9fdecfdcca86bb7cd2b98", container="metrics", endpoint="metrics", instance="10.233.178.160:9216", job="zenko/data-db-mongodb-sharded-shard0-data", member_idx="data-db-mongodb-sharded-shard0-data-1.data-db-mongodb-sharded-headless.zenko.svc.cluster.local:27017", member_state="SECONDARY", namespace="zenko", pod="data-db-mongodb-sharded-shard0-data-0", prometheus="metalk8s-monitoring/prometheus-operator-prometheus", rs_nm="data-db-mongodb-sharded-shard-0", rs_state="1"}
+        values: 2x10
+      - series: mongodb_rs_members_state{cl_id="65d9fdecfdcca86bb7cd2b98", container="metrics", endpoint="metrics", instance="10.233.178.160:9216", job="zenko/data-db-mongodb-sharded-shard0-data", member_idx="data-db-mongodb-sharded-shard0-data-2.data-db-mongodb-sharded-headless.zenko.svc.cluster.local:27017", member_state="SECONDARY", namespace="zenko", pod="data-db-mongodb-sharded-shard0-data-0", prometheus="metalk8s-monitoring/prometheus-operator-prometheus", rs_nm="data-db-mongodb-sharded-shard-0", rs_state="1"}
```

Comment:
for clarity, remove the labels we do not use (and basically ignore) → focus on the relevant labels